priv/bin/pre_start.sh

```diff
@@ -10,6 +10,9 @@ if [ -z "$CONFORM_SCHEMA_PATH" ]; then
     fi
 fi
 
+echo "[DEBUG] schema path"
```

Comment:
Can we remove this, or put it under a check for the presence of the `DEBUG_BOOT` env var which is set by the release boot script? I'd rather not have extraneous output unless it's necessary.